pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -53,9 +53,7 @@ func (p *Pilot) Hooks() *hook.Hooks {
 }
 
 func (p *Pilot) CmdFunc(pilot *v1alpha1.Pilot) (*exec.Cmd, error) {
-	// The /run.sh script is unique to gcr.io/google-samples/cassandra:v12.
-	// TODO: Add support for other Cassandra images with different entry points.
-	cmd := exec.Command("/run.sh")
+	cmd := exec.Command("/docker-entrypoint.sh")
```

Comment:
I guess we’ll still have issues handling SIGTERM with this? Unless they have properly set up signal handling in this script.